pkg/gui/context/commit_message_context.go

```diff
@@ -0,0 +1,13 @@
+package context
+
+type CommitMessageContext struct {
```

Comment:
we have an existing pattern of capturing a few things in a context, including the window and view (in this case the window will just be "commitSummary" i.e. the same as the view name, and the view will be gui.Views.CommitSummary.

See `pkg/gui/context/merge_conflicts_context.go` for an example